packages/api/db/tables.sql

```diff
@@ -87,6 +88,7 @@ CREATE TABLE IF NOT EXISTS public.user_tag
   tag             user_tag_type                                                 NOT NULL,
   value           TEXT                                                          NOT NULL,
   value_type      user_tag_value_type                                           NOT NULL,
+  reason          TEXT                                                          NOT NULL,
   inserted_at TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now())     NOT NULL,
   deleted_at  TIMESTAMP WITH TIME ZONE
 );
```

Comment:
I know is not related to this PR, but can we add a `UNIQUE (user_id, tag)` ?

Reply:
done!  great feedback!

Reply:
We can't do this because of the `deleted_at`. We need:

```
CREATE UNIQUE INDEX name_1_idx ON user_tag (user_id, tag, deleted_at)
WHERE deleted_at IS NOT NULL;

CREATE UNIQUE INDEX name_2_ix ON user_tag (user_id, tag)
WHERE deleted_at IS NULL;
```

Please rename the indexes per the name convention we use in the others

Reply:
apologies, I forgot the `deleted_at` 🤦🏼